support-workers/src/main/scala/com/gu/support/workers/lambdas/GetPaymentMethod.scala

```diff
@@ -0,0 +1,81 @@
+package com.gu.support.workers.lambdas
+
+import java.util.UUID
+
+import com.amazonaws.services.lambda.runtime.Context
+import com.gu.i18n.{Country, CountryGroup}
+import com.gu.services.{ServiceProvider, Services}
+import com.gu.support.workers._
+import com.gu.support.workers.states.{GetPaymentMethodState, CreateZuoraSubscriptionState}
+import com.gu.support.zuora.api.response.{GetPaymentMethodCardReferenceResponse, GetPaymentMethodResponse}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class GetPaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
+    extends ServicesHandler[GetPaymentMethodState, CreateZuoraSubscriptionState](servicesProvider) {
+
+  def this() = this(ServiceProvider)
+
+  override protected def servicesHandler(state: GetPaymentMethodState, requestInfo: RequestInfo, context: Context, services: Services) = {
+
+    val zuoraService = services.zuoraService
+    for {
+      account <- zuoraService.getObjectAccount(state.paymentFields.billingAccountId)
```

Comment:
Is there any check we should do to verify this account belongs to the user in state.user ?